plugins/cros-ec/fu-cros-ec-usb-device.c

```diff
@@ -532,6 +754,17 @@ fu_cros_ec_usb_device_write_firmware (FuDevice *device,
 		return FALSE;
 	}
 
+	if (self->in_bootloader) {
+		fu_device_set_custom_flags (device, "rw_written");
+	} else if (fu_device_has_custom_flag (device, "rw_written")) {
+		fu_device_set_custom_flags (device, "ro_written,rw_written");
```

Comment:
I think the convention for custom flags are dashes, not underscores.

Reply:
Done. Thanks.